sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentServiceRequest.java

```diff
@@ -1038,6 +1044,17 @@ public void dispose() {
         }
     }
 
+    private static String getThroughputControlGroupName(Object options) {
+        if (options == null) {
+            return null;
+        } else if (options instanceof RequestOptions) {
+            return ((RequestOptions) options).getThroughputControlGroupName();
+        } else {
+            // TODO: add for query and changeFeed
```

Comment:
Are you planning to add this short-term? If not we should add a GitHUb issue reference so that we keep track of it

Reply:
Plan to add in the next PR.
Added GitHub issue for tracking it.